src/App.tsx

```diff
@@ -26,70 +27,133 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  visibleGoods.sort((fistGood, secondGood) => {
+    switch (sortType) {
+      case SortType.ALPHABET:
+        return fistGood.localeCompare(secondGood);
+
+      case SortType.LENGTH:
+        return Number(fistGood.length) - Number(secondGood.length);
+
+      default:
+        return 0;
+    }
+  });
```

Comment:
Don't start sort if there is no sort type + use switch case before sort, not iside
```suggestion
  switch (sortType) {
    case SortType.ALPHABET:
      visibleGoods.sort((fistGood, secondGood) => (
        fistGood.localeCompare(secondGood);
      )

      case ....
```